src/Nest/CommonAbstractions/ConnectionSettings/ConnectionSettingsBase.cs

```diff
@@ -215,6 +218,7 @@ public TConnectionSettings MapIdPropertyFor<TDocument>(Expression<Func<TDocument
         /// <typeparam name="TDocument">The type of the document.</typeparam>
         /// <param name="propertiesSelector">The properties selector.</param>
         /// <returns></returns>
+        [Obsolete("Removed in 6.0.")]
```

Comment:
I'm inclined not to `Obsolete` this as it is likely to be noisy for a lot of users, since it is a commonly used method. Perhaps an XML comment instead?

Reply:
Moved to ///remarks